activity/views.py

```diff
@@ -0,0 +1,11 @@
+from django.shortcuts import render
+
+
+def index(request):
+    org_name = open('org_name.txt').readline()
+    return render(request, 'activity.html', {'org_name': org_name})
+
+
+def return_org_name(request):
```

Comment:
I don't see this getting used anywhere 😕

Reply:
removed.